Tasks/NugetPackager/task.json

```diff
@@ -0,0 +1,66 @@
+{
+    "id": "333b11bd-d341-40d9-afcf-b32d5ce6f24b",
+    "name": "NuGetPackager",
+    "friendlyName": "NuGet Packager and Uploader",
+    "description": "Installs and updates missing NuGet packages",
+    "helpMarkDown": "[More Information](http://go.microsoft.com/fwlink/?LinkID=613747)",
+    "category": "Package",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 17
+    },
+    "minimumAgentVersion": "1.83.0",
+    "groups": [
+        {
+            "name":"advanced",
+            "displayName":"Advanced",
+            "isExpanded":false
+        }
+    ],    
+    "instanceNameFormat": "NuGet restore $(solution)",
+    "inputs": [
+        {
+            "name": "nuspecFile",
+            "type": "filePath",
+            "label": "Path to nuspec file",
+            "defaultValue":"def.nuspec",
+            "helpMarkDown": "The path to the nuspec file",
+            "required":true
+        },
+        {
+            "name": "nugetServer",
```

Comment:
You should use a generic connected service here instead of a separate input for the server url and the api key.  This would make it easier to have multiple build definitions push to the same server without duplicating the data all over

https://github.com/Microsoft/vso-agent-tasks/blob/master/Tasks/SonarQubePreBuild/task.json#L70
https://github.com/Microsoft/vso-agent-tasks/blob/master/Tasks/SonarQubePreBuild/SonarQubePreBuild.ps1#L29